params/params_get_kwargs.v

```
@@ -34,12 +35,12 @@ pub fn (params &Params) get_default(key string, defval string) !string {
 // description is a kwarg
 pub fn (params &Params) get_int(key string) !int {
 	valuestr := params.get(key)!
-	return valuestr.int()
+	return strconv.atoi(valuestr)
```

Could you return a different error when it failed parsing? Because I just tried on my machine and the error returned by atoi is: strconv.atoi: parsing "mqlkjd": invalid syntax
Maybe thrown an error saying: "Parameter ${key} = ${valuestr} is not a valid integer"